test/config_SUITE.erl

```diff
@@ -0,0 +1,173 @@
+-module(config_SUITE).
```

Comment:
Do not mix CT tests with proper ones. Put the first ones into the SUITE files and the latter ones to the prop_* files.